skyportal/handlers/api/moving_object.py

```diff
@@ -0,0 +1,175 @@
+import traceback
+
+import arrow
+import sqlalchemy as sa
+
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+
+from ...models import (
+    DBSession,
+    Instrument,
+)
+from ...utils.moving_objects import (
+    add_instrument_fields,
+    find_obj,
+    find_observable_sequence,
+    get_ephemeris,
+)
+from ..base import BaseHandler
+
+_, cfg = load_env()
+
+
+class MovingObjectFollowupHandler(BaseHandler):
+    @auth_or_token
+    def post(self, obj_name):
+        """
+        ---
+        summary: Find a continuous sequence of observations for a moving object
+        description: Find a continuous sequence of observations for a moving object in an instrument's field
+        tags:
+        - moving objects
+        - follow-up
+        parameters:
+          - in: path
+            name: obj_name
+            required: true
+            schema:
+                type: string
+            description: Name of the moving object
+        requestBody:
+            content:
+                application/json:
+                    schema:
+                        type: object
+                        properties:
+                            instrument_id:
+                                type: integer
+                                description: ID of the instrument to use
+                            exposure_count:
+                                type: integer
+                                description: Number of exposures
+                            exposure_time:
+                                type: number
+                                description: Exposure time in seconds
+                            start_time:
```

Comment:
Maybe make it clear this is a time window that the code will attempt to fit the exposures (specified by exposure_time and exposure_count) in this window.